test/functional/test_framework/key.py

```diff
@@ -60,6 +60,33 @@ def modsqrt(a, p):
         return sqrt
     return None
 
+SECP256K1_FIELD_SIZE = 2**256 - 2**32 - 977
+
+class fe:
+    """Prime field over 2^256 - 2^32 - 977"""
+    p = SECP256K1_FIELD_SIZE
```

Comment:
`p` isn't needed anymore; you can also use `SECP256K1_FIELD_SIZE` directly.

Reply:
Done.